WMass/python/plotter/w-helicity-13TeV/wmass_e/mca-includes/mca-80X-wenu-sig.txt

```diff
@@ -1,8 +1,17 @@
 # signal from the weighted uncut sample
-Wplus_long  : WJetsToLNu_NLO* : 3.*20508.9    : genw_decayId == 12 && genw_charge>0 ; FillColor=ROOT.kGray+1   , FakeRate="w-helicity-13TeV/fractionReweighting/reweighting_0_plus.txt", Label="W+ long"
-Wplus_left  : WJetsToLNu_NLO* : 3.*20508.9    : genw_decayId == 12 && genw_charge>0 ; FillColor=ROOT.kBlue-1   , FakeRate="w-helicity-13TeV/fractionReweighting/reweighting_L_plus.txt", Label="W+ left"
-Wplus_right : WJetsToLNu_NLO* : 3.*20508.9    : genw_decayId == 12 && genw_charge>0 ; FillColor=ROOT.kGreen+1  , FakeRate="w-helicity-13TeV/fractionReweighting/reweighting_R_plus.txt", Label="W+ right"
+Wplus_long  : WJetsToLNu_LO* : 3.*20508.9    : genw_decayId == 12 && genw_charge>0 ; FillColor=ROOT.kGray+1   , FakeRate="w-helicity-13TeV/fractionReweighting/reweighting_0_plus.txt", Label="W+ long"
+Wplus_left  : WJetsToLNu_LO* : 3.*20508.9    : genw_decayId == 12 && genw_charge>0 ; FillColor=ROOT.kBlue-1   , FakeRate="w-helicity-13TeV/fractionReweighting/reweighting_L_plus.txt", Label="W+ left"
+Wplus_right : WJetsToLNu_LO* : 3.*20508.9    : genw_decayId == 12 && genw_charge>0 ; FillColor=ROOT.kGreen+1  , FakeRate="w-helicity-13TeV/fractionReweighting/reweighting_R_plus.txt", Label="W+ right"
```

Comment:
do not change these: the default should be NLO samples (we should update the efficiency file as well, now they were computed with the LO)